### DISCUSSION TITLE
When querying dynamodb how do  I get the results as JSON

### DISCUSSION THREAD

**Discussion by Ancient-Dragon** [aws/aws-sdk-java-v2#3690] [Q&A]
Apologies these seems like a really basic question but I could not find any documentation on how to use the enhanced query to query for multiple items and bring the data back as JSON rather than having to type out a class?

Any help would be greatly appreciated!

**Comment by debora-ito** [comment]
This is not supported currently. 

We do have a feature request to add a JSON attribute converter to the list of available converters by default (https://github.com/aws/aws-sdk-java-v2/issues/2162), but another option is to write a custom converter and you can associate with the attribute in your table schema.
